### DISCUSSION TITLE
See my container application in the camera UI

### DISCUSSION THREAD

**Discussion by sebaxis** [AxisCommunications/discussions#2] [ACAP] [answered]
Hello,

I am used to see my installed application (.eap) from the camera UI. How can I achieve that from my container application?

Thanks

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
We have [an existing example](https://github.com/AxisCommunications/acap-native-sdk-examples/tree/master/container-example) where a containerised application is built into an .eap file in the [Native SDK example repository](https://github.com/AxisCommunications/acap-native-sdk-examples).

The Docker image is bundled within the .eap file, thus in a sense becoming self-contained. We recognise that this isn't desired in all situations and will probably look into an example where Docker images are pulled from a registry when the application is started. When we have that example I'll let you know.

Does this sound ok to you?
